reconcile/utils/vcs.py

```diff
@@ -222,9 +222,11 @@ def get_file_content_from_app_interface_master(self, file_path: str) -> str:
             if not file_path.startswith("data")
             else file_path
         )
-        return self._app_interface_api.project.files.get(
-            file_path=file_path, ref="master"
-        ).decode()
+        return (
+            self._app_interface_api.project.files.get(file_path=file_path, ref="master")
+            .decode()
+            .decode("utf-8")
```

Comment:
this a breaking change, the first `decode()` returns `byte`, now this method returns `str`, was `byte` expected before?